### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by danialnurhakim**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by danialnurhakim — COMMENTED** [review]
Good concise change for better understanding of player.

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (danialnurhakim) [inline index.html]: Cool.

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
...
 
 <script>
```
Comment (dnlnrkm) [inline index.html:52]:
```suggestion
<script>
```

File: index.html (hunk 1 above)
Comment (danialnurhakim) [inline index.html]:
```suggestion
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖! (V2)</h2>
```

**Pull request merged**